x/lockup/keeper/synthetic_lock.go

```diff
@@ -141,21 +157,6 @@ func (k Keeper) CreateSyntheticLockup(ctx sdk.Context, lockID uint64, synthDenom
 	return nil
 }
 
-// DeleteAllSyntheticLocks iterates over given array of synthetic locks and deletes all individual synthetic locks.
-func (k Keeper) DeleteAllSyntheticLocks(ctx sdk.Context, lock types.PeriodLock, synthLocks []types.SyntheticLock) error {
-	if len(synthLocks) == 0 {
-		return nil
-	}
-
-	for _, synthLock := range synthLocks {
-		err := k.DeleteSyntheticLockup(ctx, lock.ID, synthLock.SynthDenom)
-		if err != nil {
-			return err
-		}
-	}
-	return nil
-}
```

Comment:
No longer needed. We use DeleteSyntheticLockup directly now since we never expect multiple synthLocks anymore